nexus-repository-p2-it/pom.xml

```diff
@@ -77,6 +77,17 @@
       <version>${nxrm-version}</version>
       <type>zip</type>
       <scope>test</scope>
+      <exclusions>
+        <!-- Avoid leaking older r plugins from distro -->
```

Comment:
should be p2 instead of r

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/68/commits/cf8bef1e36ef43e71039b05d9f422becfd5477b3